dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/outlierdetection/AnalyticsOutlierDetectionController.java

```diff
@@ -144,4 +156,11 @@ private OutlierRequest getFromQuery(OutlierQueryParams queryParams, boolean anal
 
     return request;
   }
+
+  private void checkAnalyticsTableForOutliers() {
```

Comment:
Would be nice to have a e2e to test this error. Please see the class `NoAnalyticsTablesErrorsScenariosTest`. You can add a new test there to simulate this exception and assert the response/error.

Reply:
e2e is coming, there are no e2e for outliers

Reply:
Okay, if you prefer to add it as part of all others e2e for Outliers is fine. Just please add a TODO in the class `NoAnalyticsTablesErrorsScenariosTest`, so we do not forget this scenario.

Reply:
done